tests/unittest.py

```diff
@@ -310,15 +310,21 @@ def setUp(self):
                     )
                 )
 
-                async def get_user_by_access_token(token=None, allow_guest=False):
+                async def get_user_by_access_token(
+                    token: Optional[str] = None, allow_guest: bool = False
+                ) -> JsonDict:
                     assert self.helper.auth_user_id is not None
                     return {
                         "user": UserID.from_string(self.helper.auth_user_id),
                         "token_id": token_id,
                         "is_guest": False,
                     }
 
-                async def get_user_by_req(request, allow_guest=False):
+                async def get_user_by_req(
+                    request: SynapseRequest,
+                    allow_guest: bool = False,
+                    allow_expired: bool = False,
```

Comment:
This argument is new---is this a fix to match the signature of the real `get_user_by_req`?

Reply:
> This argument is new---is this a fix to match the signature of the real `get_user_by_req`?

Yes, should have specified, sorry! 👍